client/pages/students.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from 'react'
+
+// TODO(amantri): get this from the server code, preferably auto-generated
+interface Student {
+  student_id: string;
+  first_name: string;
+  last_name: string;
+  assigned_case_manager: string;
+}
+
+export default function Students() {
+  const [data, setData] = useState(null)
+  const [isLoading, setLoading] = useState(false)
+
+  useEffect(() => {
+    setLoading(true)
+    // TODO(amantri): Get backend url from env
+    fetch('http://localhost:8080/students/list')
+      .then((res) => res.json())
+      .then((data) => {
+        setData(data)
+        setLoading(false)
+      })
+  }, [])
+
+  if (isLoading) return <p>Loading...</p>
+  if (!data) return <p>No profile data</p>
+
+  return (
+    <>
+      <title>Student list</title>
+
+      <div>
```

Comment:
The current table is quite ugly, but I didn't want to spend too much time fighting the css quite yet :)